dorm/dorm/Presenter/RoomManager/RoomManagerViewController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RoomManagerViewController.swift
+//  dorm
+//
+//  Created by JongHo Park on 2022/07/19.
+//
+
+import UIKit
+
+/// RoomManagerViewController - Room Collection View 와 Room Detail Controller 를 관리하는 ViewController 입니다.
+final class RoomManagerViewController: UIViewController {
+
+    private lazy var roomManagerView: RoomManagerView = RoomManagerView()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func loadView() {
```

Comment:
```suggestion
    override func loadView() {
        super.loadView()
```

override method에서는 부모 class의 실행이 되도록 super로 불러주는 것이 중요해요 !

Reply:
4dadabb1e8cf77756c4229437ffaf0c0139698bf
에서 수정했습니다!